libphodav/chezdav.c

```
@@ -228,12 +232,21 @@ main (int argc, char *argv[])
     my_error (_ ("option parsing failed: %s\n"), error->message);
   g_option_context_free (context);
 
+  if (argc != 1)
+    my_error (_ ("unsupported extra arguments: %s ...\n"), argv[1]);
```

i fixed this bug on the way...
extra arguments were silently ignored.